integration-tests/tzip16-originate-wallets-with-metadata-on-HTTPS-and-sha256-hash-and-fetch-metadata.spec.ts

```diff
@@ -0,0 +1,200 @@
+import { CONFIGS } from "./config";
```

Comment:
The content of this file seems identical to `integration-tests/tzip16-originate-wallets-with-metadata-on-HTTPS-and-fetch-metadata.spec.ts `

Reply:
ok, the test already calls to wallets in them. So when I converted I got duplication. Thanks. I will remove this file.